style/app.scss

```diff
@@ -687,131 +642,121 @@ a:active
 
 #sidebar
 {
-  display          : inline-block;
-  width            : 40px;
-  height           : 100%;
-  float            : left;
-  background-color : #7a6a96;
-  position         : absolute;
-  padding-left     : 23px;
+  display          : flex;
+  flex-direction   : column;
+  z-index          : 1;
+  position         : inherit;
+  width            : 140px;
```

Comment:
Overall the box is a bit small for my taste. Make this wider, and make the font for the contents be 14px rather than 12px.